cmd/load-generator/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/letsencrypt/boulder/cmd"
+)
+
+type Config struct {
+	// Execution plan parameters
+	Plan struct {
+		Actions   []string // things to do
+		Rate      int64    // requests / s
+		RateDelta string   // requests / s^2
+		MaxRate   int64    // XXX: is this needed?
+		Runtime   string   // how long to run for
+	}
+	ExternalState string   // path to file to load/save registrations etc to/from
+	DontSaveState bool     // don't save changes to external state
+	APIBase       string   // ACME API address to send requests to
+	DomainBase    string   // base domain name to create authorizations for
+	ChallTypes    []string // which challenges to complete, empty means use all
+	HTTPOneAddr   string   // address to listen for http-01 validation requests on
+	TLSOneAddr    string   // address to listen for tls-sni-01 validation requests on
+	RealIP        string   // value of the Real-IP header to use when bypassing CDN
+	RegKeySize    int      // size of the key to use in registrations
```

Comment:
I don't think this is used anywhere - it looks like the registration key is always a P256 ecdsa key.